src/main/java/com/puppycrawl/tools/checkstyle/utils/XpathUtil.java

```diff
@@ -106,7 +106,7 @@
         Stream.of(
             TokenTypes.IDENT, TokenTypes.STRING_LITERAL, TokenTypes.CHAR_LITERAL,
             TokenTypes.NUM_LONG, TokenTypes.NUM_INT, TokenTypes.NUM_DOUBLE, TokenTypes.NUM_FLOAT,
-            TokenTypes.TEXT_BLOCK_CONTENT)
+            TokenTypes.TEXT_BLOCK_CONTENT, TokenTypes.COMMENT_CONTENT)
```

Comment:
also do not forget to add tests in XpathMapperTest,

to demonstrate that AST node can be queried by comment content

Reply:
Sir, can you confirm if `SINGLE_LINE_COMMENT` can be queried because I am not able to get it `SINGLE_LINE_COMMENT` is being skipped

log:
```
$ cat InputXpathMapperSingleLineComment.java
package com.puppycrawl.tools.checkstyle.xpath.xpathmapper;

public class InputXpathMapperSingleLineComment {
    int num; // some comment
}

$ java -jar checkstyle-8.41.1-all.jar -J InputXpathMapperSingleLineComment.java
PACKAGE_DEF -> package [1:0]
|--ANNOTATIONS -> ANNOTATIONS [1:45]
|--DOT -> . [1:45]
|   |--DOT -> . [1:39]
|   |   |--DOT -> . [1:28]
|   |   |   |--DOT -> . [1:22]
|   |   |   |   |--DOT -> . [1:11]
|   |   |   |   |   |--IDENT -> com [1:8]
|   |   |   |   |   `--IDENT -> puppycrawl [1:12]
|   |   |   |   `--IDENT -> tools [1:23]
|   |   |   `--IDENT -> checkstyle [1:29]
|   |   `--IDENT -> xpath [1:40]
|   `--IDENT -> xpathmapper [1:46]
`--SEMI -> ; [1:57]
CLASS_DEF -> CLASS_DEF [3:0]
|--MODIFIERS -> MODIFIERS [3:0]
|   `--LITERAL_PUBLIC -> public [3:0]
|--LITERAL_CLASS -> class [3:7]
|--IDENT -> InputXpathMapperSingleLineComment [3:13]
`--OBJBLOCK -> OBJBLOCK [3:47]
    |--LCURLY -> { [3:47]
    |--VARIABLE_DEF -> VARIABLE_DEF [4:4]
    |   |--MODIFIERS -> MODIFIERS [4:4]
    |   |--TYPE -> TYPE [4:4]
    |   |   `--LITERAL_INT -> int [4:4]
    |   |--IDENT -> num [4:8]
    |   `--SEMI -> ; [4:11]
    |--SINGLE_LINE_COMMENT -> // [4:13]
    |   `--COMMENT_CONTENT ->  some comment\n [4:15]
    `--RCURLY -> } [5:0]
 ```
 
 But when I try to get the second last element of `OBJBLOCK`
 `Expected: [<VARIABLE_DEF[4x2]>]` and `.findFirstToken(TokenTypes.SINGLE_LINE_COMMENT)` returns Null pointer exception.